gradle-pride-plugin/src/main/resources/META-INF/gradle-plugins/com.prezi.pride.properties

```diff
@@ -0,0 +1 @@
+implementation-class=com.prezi.gradle.pride.PridePlugin
```

Comment:
I changed the group ID and the plugin ID, but not the package prefix. I can do that too, if you'd like (though it's not strictly necessary).